packages/i18n/src/async-locale-data/async-locale-data.spec.tsx

```diff
@@ -0,0 +1,115 @@
+import { mocked } from 'ts-jest/utils';
```

Comment:
I found this here: https://github.com/kulshekhar/ts-jest/issues/576#issuecomment-458178545

It's basically to have the mocked modules `jest.mock` properly typed.